go/oasis-net-runner/fixtures/default.go

```diff
@@ -17,14 +21,16 @@ import (
 )
 
 const (
+	cfgEpochtimeMock       = "net.epochtime_mock"
+	cfgFixtureFile         = "net.fixture.file"
```

Comment:
I would rename this to just `fixture.file` as `net.*` are used to populate the default fixture.

Maybe `net.*` should be renamed to `fixture.default.*` to be more clear?